--- env/lib/python3.6/site-packages/zds_client/client.py
+++ env/lib/python3.6/site-packages/zds_client/client.py
@@ -195,6 +195,8 @@
         :raises: :class:`requests.HTTPException` for internal server errors
         :raises: :class:``ClientError` for HTTP 4xx status codes
         """
+        if not path.startswith(f'/{self.service}'):
+            path = '/' + self.service + path
         url = urljoin(self.base_url, path)

         headers = kwargs.pop('headers', {})